lib/blueprinter/extractors/association_extractor.rb

```diff
@@ -6,14 +6,18 @@ def initialize
 
     def extract(association_name, object, local_options, options={})
       value = @extractor.extract(association_name, object, local_options, options.except(:default))
-      return options[:default] if value.nil?
+      return default_value(options) if value.nil?
       view = options[:view] || :default
       blueprint = association_blueprint(options[:blueprint], value)
       blueprint.prepare(value, view_name: view, local_options: local_options)
     end
-    
+
     private
 
+    def default_value(association_options)
+      association_options.key?(:default) ? association_options.fetch(:default) : Blueprinter.configuration.association_default
```

Comment:
Rather than this ternary, I would prefer something like
```suggestion
      assosiation_options.fetch(:default, Blueprinter.configuration.association_default)
```

What do you think?

Reply:
I think that makes sense to me, but I avoided this initially, because I literally wanted to do a check on whether the `:default` key was present in the options in case it was explicitly set to nil, we'd want to actually render nil instead of pass through to the global Blueprinter default.
Make sense? I don't know if that's an important enough behavior to preserve or not. Thoughts? 🤔

Reply:
Yeah that totally makes sense. I could see someone switching it up like that if only one or two endpoints ignore the default and use nil instead.